docs/DEPLOYMENT.md

```diff
@@ -20,27 +23,99 @@
 - Kubernetes Cluster
   - You can also deploy on [Minkube][minikube], [kind][kind] or [OpenShift][openshift].
 - [kubectl][kubectl]
-- [ko][ko]
+- [docker][docker] or [podman][podman]
 
-  - You can find installation steps [here][ko] or if you have Go installed then you can execute the command :-
+## Build Images
 
-    ```bash
-    go get github.com/google/ko/cmd/ko
-    ```
+Lets build and push all images to a registry and then create deployments.
 
-- [docker][docker] or [podman][podman]
+### UI Image
 
-## Deploy API Service
+Ensure you are in `ui` directory and logged in you image registry.
+
+```
+docker build -t <image> . && docker push <image>
+```
+
+Replace `<image>` with the registry and image name. 
+eg. `quay.io/<username>/ui`
+
+### API & DB Migration Image
+
+Ensure you are in `api` directory and logged in you image registry.
+
+Build the API image using below command
+```
+docker build -t <image> . && docker push <image>
+```
+
+Replace `<image>` with the registry and image name. 
+eg. `quay.io/<username>/api`
+
+Now, Build the Db migration image using below command
+```
+docker build -f db.Dockerfile -t <image> . && docker push <image>
+```
+
+Replace `<image>` with the registry and image name. 
+eg. `quay.io/<username>/db-migration`
+
+Make sure all images are public before creating deployments.
+
+## Deploy Database
+
+Now, we have all images pushed to registry. Lets deploy the database first.
+
+Navigate to `config` directoy in the root of the project.
 
-Ensure you are in `api` directory
+Execute below command
+
+```
+kubectl apply -f 00-init/
+```
+This will create `tekton-hub` namespace, db deployment with `postgres` image, secret to save db credentials, pvc for the db and a clusterip service to expose the db internally.
```

Comment:
nit (feel free to add only there is major change)
```suggestion
This will create `tekton-hub` namespace, db deployment with `postgres` image, secret to save db credentials, pvc for the db and a ClusterIP service to expose the db internally.
```